mslib/mscolab/mscolab.py

```diff
@@ -130,6 +400,13 @@ def main():
         print("Version:", __version__)
         sys.exit()
 
+    try:
+        _ = git.Repo(os.path.dirname(os.path.realpath(__file__)), search_parent_directories=True)
```

Comment:
I am not sure that this is a reasonable use case. Useful for testing, but adds a  lot of code that needs to be maintained.